packages/video_player/video_player/lib/video_player.dart

```diff
@@ -905,16 +906,16 @@ class ClosedCaption extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
+    if (text == null || text!.isEmpty) {
```

Comment:
It'd make sense to do:

```dart
final text = this.text;
if (text == null || text.isEmpty) {
}
```
(and remove the `!` on `Text` too)

That's a bit cheaper

Reply:
That is what I wrote before. But @Salakar just removed that

Reply:
Yep my bad, reverting